spyder/utils/tests/test_programs.py

```diff
@@ -132,5 +133,11 @@ def test_get_temp_dir_ensure_dir_exists():
     assert another_call == temp_dir
 
 
+def test_is_stable_version():
+    """Tests for is_stable_version."""
+    assert is_stable_version('3.3.0')
+    assert not is_stable_version('4.0.0b1')
```

Comment:
For completeness, would you want to check ``'3.3.2.dev0`` as well, since its of a slightly different form?